core/components/molecules/pager/pager.js

```diff
@@ -1,47 +1,69 @@
-import React from 'react'
-import PropTypes from 'prop-types'
-import styled from 'styled-components'
-import Button from '../../atoms/button'
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
 
-import { changePageIfAppropiate, totals } from '../../_helpers/pagination'
+import Button from "../../atoms/button";
+import Icon from "../../atoms/icon";
+import { changePageIfAppropiate, totals } from "../../_helpers/pagination";
 
 const StyledPager = styled.div`
   display: flex;
   justify-content: space-between;
   align-items: center;
-`
+`;
 
 const StyledPageSelector = styled.div`
   display: flex;
   flex-direction: row;
   align-items: center;
-`
+`;
+
+const StyledButton = styled(Button)`
+  ${props =>
+    props.position === "left" ? "padding-right" : "padding-left"}: 11px;
+  ${props =>
+    props.position === "left" ? "padding-left" : "padding-right"}: 7px;
+
```

Comment:
I'm not super in the context, why do we need to modify the Button?

Reply:
The icon does not play well with the Button as-is. So I had to adjust the padding.